index.html

```diff
@@ -1,11 +1,13 @@
 ---
 permalink: /
 title: Glosario
+layout: page
 ---
 {% for lang in site.languages %}
   {% assign link = lang.key | prepend: './' | append: '/' | relative_url %}
-  <h2><a href="{{link}}">{{lang.name}}</a></h2>
-  <div dir='auto'>
+  <h2><a id="{{lang.name}}" href="{{link}}">{{lang.name}}</a></h2>
+  <div dir='auto' width="60%">
```

Comment:
my understanding is that it's better to specify width using CSS

Reply:
I have added a `.lang_blurb` class to the CSS file for this.